CMakeLists.txt

```diff
@@ -88,8 +88,8 @@ add_library(trajectories)
 set_target_properties(trajectories PROPERTIES SYSTEM TRUE)
 target_sources(trajectories
   PRIVATE
-    trajectories/Trajectory.cpp
-    trajectories/Path.cpp
+    src/third_party/trajectories/Trajectory.cpp
+    src/third_party/trajectories/Path.cpp
```

Comment:
I think the better solution will be creating a `CMakeLists.txt` in `src/third_party` that can manage all those, though not familiar enough with CMake yet to make these optimizations